README.md

```diff
@@ -51,9 +51,9 @@ When master has accumulated changes you want to release, please perform the foll
 - update the [build.gradle](build.gradle)'s `appVersion` accordingly
 - if you want to release a new version of the Gradle plugin:
     - update `BuildVersion.kt`
-    - run `./gradlew deployGradlePlugin`
 - commit and push your changes
-- create a GitHub Release tag with the same version number and the text from the changleog.
+- create a GitHub Release tag with the same version number and the text from the changelog.
+- After the build finished visit https://oss.sonatype.org/#stagingRepositories, close and release the staging repository
```

Comment:
how do I log in there? do I need some special account? log in with google? etc.

Reply:
The credentials are stored in the teamscale.kdbx keystore

Reply:
Can you please add this as a comment here, so future maintainers don't have to ask (and I can look it up when I eventually forget in 2 months :P)

Reply:
Done ;-)